sequencer/addrqueue.go

```diff
@@ -78,6 +80,11 @@ func (a *addrQueue) addTx(tx *TxTracker) (newReadyTx, prevReadyTx, replacedTx *T
 	}
 }
 
+// addForcedTX adds a forced tx to the list of forced txs
+func (a *addrQueue) addForcedTX(txHash common.Hash) {
```

Comment:
addForcedTx (x lowercase to be consistent with addTx and deleteForcedTx function)